app/models/concerns/control_objectives/scopes.rb

```diff
@@ -15,5 +15,17 @@ def default_order
         POSTGRESQL_ADAPTER ? { name: :asc } : { created_at: :asc }
       )
     end
+
+    def visible
+      setting = Current.organization.settings.find_by name: 'hide_obsolete_best_practices'
+
+      hide_obsolete_best_practices = DEFAULT_SETTINGS[:hide_obsolete_best_practices][:value]
+
+      if (setting ? setting.value : hide_obsolete_best_practices) == '0'
+        all
+      else
+        where(obsolete: false )
```

Comment:
```suggestion
        where(obsolete: false)
```

**APLICÁ LOS SUGGESTIONS O PONELOS ENANO, TENGO QUE HACER VARIOS MÁS DE UNA VEZ.**